packages/assets/test/assets.tests.ts

```diff
@@ -2,6 +2,9 @@ import { Assets } from '@pixi/assets';
 import { BaseTexture, Texture } from '@pixi/core';
 import { Spritesheet } from '@pixi/spritesheet';
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const keepImportAlive = [Spritesheet];
```

Comment:
cc @bigtimebuddy - not sure if there is a more elegant way!

Reply:
Does just doing this work?

`import '@pixi/spritesheet'`